### ISSUE TITLE
children prop doesn't exist in HtmlRenderer

### ISSUE THREAD

**Issue by chathu-novade** [danomatic/react-pdf-html#37]
HtmlRenderer doesn't have the types for children prop

**Comment by danomatic** [comment]
I don't think it needs it explicitly defined because:

```ts
export type HtmlRenderer = React.FC<{
  element: HtmlElement;
  style: Style[];
  stylesheets: HtmlStyles[];
}>;
```

in react types...
```ts
    type FC<P = {}> = FunctionComponent<P>;

    interface FunctionComponent<P = {}> {
        (props: PropsWithChildren<P>, context?: any): ReactElement<any, any> | null;
        propTypes?: WeakValidationMap<P> | undefined;
        contextTypes?: ValidationMap<any> | undefined;
        defaultProps?: Partial<P> | undefined;
        displayName?: string | undefined;
    }

    type PropsWithChildren<P> = P & { children?: ReactNode | undefined };
```

**Comment by chathu-novade** [comment]
I'm using  "@types/react": "^18.0.15" and this seems a problem with it.

**Comment by chathu-novade** [comment]
@danomatic Seems We needs to use React.FC<React.PropsWithChildren<Props>> with @types/react 18

**Comment by danomatic** [comment]
@chathu-novade this should be fixed in version `1.1.10`
